frontend/src/setup/components/DatabaseStep.jsx

```diff
@@ -37,25 +37,47 @@ export default class DatabaseStep extends Component {
             'formError': null
         });
 
+        // make sure that we are trying ssl db connections to start with
+        details.details.ssl = true;
+
         try {
-            // validate them first
+            // validate the details before we move forward
             await this.props.dispatch(validateDatabase(details));
 
-            // now that they are good, store them
-            this.props.dispatch(setDatabaseDetails({
-                'nextStep': ++this.props.stepNumber,
-                'details': details
-            }));
+        } catch (error) {
+            let formError = error;
+            if (details.details.ssl === true) {
```

Comment:
Is the check for `ssl === true` actually necessary? Isn't it always going to be since we explicitly set it above?